lmwrapper/structs.py

```diff
@@ -213,6 +218,9 @@ def dict_serialize(self) -> dict:
         Serialize the prompt into a json-compatible dictionary. Note this is not
         guaranteed to be the same as the JSON representation for use
         in an openai api call. This is just for serialization purposes.
+        
```

Comment:
hm actually maybe it should be a user configurable arg on whether to include the user metadata on it. We will need some reasonable behaviour for attempting to make the the user metadata a dict seralization (like if it is all simple types), or failing if it is is not.

Reply:
additionally we will need to an option pass through in LmPrediction.dict_serialize